packages/core/src/profile/redux/reducer.js

```diff
@@ -132,6 +138,7 @@ const result = (state = INITIAL_STATE.result, action = {}) => {
     case actionTypes.UPDATE_PROFILE_SUCCESS:
     case actionTypes.CREATE_GUEST_USER_SUCCESS:
     case actionTypes.GET_GUEST_USER_SUCCESS:
+    case actionTypes.GET_USER_ATTRIBUTES_SUCCESS:
```

Comment:
This is what i was referring to in my previous comment. The result of this should be in its own area.
Remove this from here.